### PR TITLE
Make extra clear that Ubuntu is not supported with Supervised

### LINKED ISSUE #21715: Home assistant supervised only on debian
### Feedback

Hello, 
I could be clearer from the installers page that the supervised edition only works on Debian.

Thank you,
The guy that just installed ubuntu server and is now reinstalling the OS xD

### URL

https://www.home-assistant.io/installation/linux/

### Version

2022.2.9

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adds an additional step to the Supervised installation guide, to make it extra clear that only Debian is supported.

It is a recurring subject and common mistake to use Ubuntu, Armbian or Raspberry Pi OS resulting in discussions or reported issues later on.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #21715

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/supervised.md [hunk 1]
```diff
@@ -9,7 +9,10 @@ Unless you really need this installation type, you should install Home Assistant
 </div>
 
 1. First make sure you understand the <a href="https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md" target="_blank">requirements</a>.
-2. Then head over to <a href="https://github.com/home-assistant/supervised-installer" target="_blank">home-assistant/supervised-installer</a> to set it up.
+2. This installation method has very strict requirements, for example, it only
+   supports Debian (and **not** Ubuntu, Armbian, Raspberry Pi OS, those are not
+   supported). So, make sure you understand the requirements from step 1 above.
```
Comment (zsarnett) [inline source/_includes/installation/supervised.md]: This is a bit weird wording at the end. with Say Not Twice in the parenthesis

Comment (zsarnett) [reply source/_includes/installation/supervised.md]:
```suggestion
   supports Debian (and Ubuntu, Armbian, Raspberry Pi OS are **not** 
   supported). So, make sure you understand the requirements from step 1 above.
```

Comment (zsarnett) [reply source/_includes/installation/supervised.md]: Maybe something like this?

Comment (frenck) [reply source/_includes/installation/supervised.md]: Thanks!

**Review by zsarnett — APPROVED** [review]

**Pull request merged**